app/_src/kubernetes-ingress-controller/deployment/minikube.md

```diff
@@ -47,6 +47,26 @@ Kubernetes cluster.
    export PROXY_IP=<ip-address>
    echo $PROXY_IP
    ```
+
+1. Verify that you can access {{site.base_gateway}}:
+
+   ```bash
+   curl -i $PROXY_IP
+   ```
 After you've installed the {{site.kic_product_name}}, please follow the
 [getting started](/kubernetes-ingress-controller/{{page.kong_version}}/guides/getting-started) tutorial to learn
 about how to use the Ingress Controller.
+
+## Troubleshooting connection issues
+
+The network is limited if you are using the Docker driver on Darwin, Windows, or WSL, and the Node IP is not reachable directly.
+When you deploy minikube on Linux with the Docker driver results in no tunnel being created. For more information, see [accessing apps](https://minikube.sigs.k8s.io/docs/handbook/accessing/#using-minikube-service-with-tunnel).
+
+### Error message
+
+`curl: (7) Failed to connect to 127.0.0.1 port 80 after 5 ms: Couldn't connect to server`
+
+### Work around
+
+ Run the `minikube tunnel` command in a separate terminal window to keep the tunnel open. 
```

Comment:
What does the tunnel do here?

Reply:
Added additional infor about what a tunnel does.